docs/docs/adding-comments.md

```diff
@@ -1,71 +1,71 @@
+No entanto, lembre-se de que a escolha do Disqus também gera um ponto negativo. Seu site não é mais totalmente estático, mas depende de uma plataforma externa para enviar seus comentários por meio de `iframe` incorporado em tempo real. Além disso, você deve considerar as implicações de privacidade ao permitir que terceiros armazenem os comentários de seus visitantes e potencialmente rastreiem o comportamento de navegação deles. Você pode consultar a [política de privacidade Disqus](https://help.disqus.com/terms-and-policies/disqus-privacy-policy), as [FAQs sobre privacidade](https://help.disqus.com/terms-and-policies/privacy-faq) (especificamente a última pergunta sobre conformidade com GDPR) e informe seus usuários [como editar suas configurações de compartilhamento de dados](https://help.disqus.com/terms-and-policies/how-to-edit-your-data-sharing-settings).
 
-If these concerns outweigh the benefits of Disqus, you may want to look into some of the other options above. We welcome pull requests to expand this guide with setup instructions for other services.
+Se essas preocupações superam os benefícios do Disqus, convém examinar algumas das outras opções acima. Nós incentivamos os _pulls requests_ para expandir este guia com instruções de configuração para outros serviços.
 
-## Implementing Disqus
+## Implementando o Disqus
 
-![Disqus logo](images/disqus-logo.svg)
+![Logo do Disqus](images/disqus-logo.svg)
 
-Here are the steps for adding Disqus comments to your own blog:
+Aqui estão as etapas para adicionar comentários do Disqus ao seu próprio blog:
 
-1. [Sign-up to Disqus](https://disqus.com/profile/signup). During the process you'll have to choose a shortname for your site. This is how Disqus will identify comments coming from your site. Copy that for later.
-2. Install the Disqus React package
+1. [Inscreva-se no Disqus](https://disqus.com/profile/signup). Durante o processo, você terá que escolher um nome abreviado para o seu site. É assim que o Disqus identificará os comentários provenientes do seu site. Copie isso para mais tarde.
+2. Instale o pacote Disqus React
 
 ```shell
 npm install disqus-react
 ```
 
-3. Add the shortname from step 1 as something like `GATSBY_DISQUS_NAME` to your `.env` and `.env.example` files so that people forking your repo will know that they need to supply this value to get comments to work. (You need to prefix the environment variable with `GATSBY_` in order to [make it available to client-side code](https://www.gatsbyjs.org/docs/environment-variables/#client-side-javascript).)
+3. Adicione o nome abreviado da etapa 1 como `GATSBY_DISQUS_NAME` nos arquivos `.env` e `.env.example` para que as pessoas que estão participando do seu repositório saberem que precisam fornecer esse valor para que os comentários funcionem. (Você precisa prefixar a variável de ambiente com `GATSBY_` para [disponibilizá-la no código do lado do cliente](https://www.gatsbyjs.org/docs/environment-variables/#client-side-javascript).)
```

Comment:
Algumas mínimas modificações que eu acham que também melhoram a fluidez to texto.

```suggestion
3. Adicione o nome abreviado da etapa 1 como `GATSBY_DISQUS_NAME` nos seus arquivos `.env` e `.env.example` para que as pessoas que estão participando do seu repositório saibam que precisam fornecer esse valor para que os comentários funcionem. (Você precisa prefixar a variável de ambiente com `GATSBY_` para [disponibilizá-la no código do lado do cliente](https://www.gatsbyjs.org/docs/environment-variables/#client-side-javascript).)
```